fiber-unix/src/scheduler.ml

```diff
@@ -167,8 +168,6 @@ let is_empty table = Table.length table = 0
 
 let me = Fiber.Var.create ()
 
-let scheduler () = Fiber.Var.get_exn me
```

Comment:
You use `Fiber.Var.get_exn me` at different places. Maybe it's worth keeping a function for it?

Reply:
I'm trying to refrain from helpers that contain no logic and are just chains of projections.